packages/gnome_terminal.rb

```diff
@@ -29,6 +31,7 @@ class Gnome_terminal < Package
   depends_on 'adobe_source_code_pro_fonts' # (Needed for monospace fonts)
   depends_on 'yelp_tools'
   depends_on 'gtk_doc'
+  depends_on 'sommelier' unless ARCH == 'i686'
```

Comment:
Does gnome terminal work without sommelier on i686?  If it doesn't then might as well exclude the i686 binary.  Sommelier is DOA on my i686 chromebook.

Reply:
nope. So I removed i686.